src/mainboard/raptor-cs/talos-2/Kconfig

```diff
@@ -27,6 +27,23 @@ config BOARD_SPECIFIC_OPTIONS
 	select IPMI_BT_ROMSTAGE
 	select FLATTENED_DEVICE_TREE
 
+config TALOS_2_INFINEON_TPM_1
+	bool "I2C SLB9635TT or SLB9645TT or another TPM1 chip compatible with SLB9635TT"
+	default n
+	select TPM_MEASURED_BOOT # needed for TCPA log
+	select MAINBOARD_HAS_I2C_TPM_GENERIC
+	select MAINBOARD_HAS_TPM1
+
+config DRIVER_TPM_I2C_BUS
+	hex "I2C TPM chip bus"
+	default 0x02
+	depends on TALOS_2_INFINEON_TPM_1
```

Comment:
No need to depend on that I guess

Reply:
This is to not show these fields unless TPM support is enabled.

Reply:
Ahh right. But then is it possible to have TPM on different I2C buses?